starboard/android/apk/app/src/main/java/dev/cobalt/media/MediaCodecUtil.java

```diff
@@ -659,8 +656,7 @@ public static String findVideoDecoder(
       // Enable the improved support check based on more specific APIs, like isSizeSupported() or
       // areSizeAndRateSupported(), for 8k content. These APIs are theoretically more accurate,
       // but we are unsure about their level of support on various Android TV platforms.
-      final boolean enableImprovedSupportCheck =
```

Comment:
Like below, `force_improved_support_check` is on by default, so |enableImprovedSupportCheck| should probably be true by default now, or removed.

Reply:
Only remove `enableImprovedSupportCheck `.